### PR TITLE
Math magicians : full website

### PR DESCRIPTION
**Pull request by mr-abdellah**
In this branch, i create a full website using : 
- React
- Css
- React Router

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <div>
+      <h4>Welcome to our page !</h4>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+      <p> Curabitur pretium convallis tortor id venenatis.</p>
+      <p> Aenean quis justo ut orci aliquam mollis. Integer condimentum a </p>
+      <p> volutpat eget tellus ut, mollis ornare tortor</p>
+    </div>
+  );
+}
```
Comment (roshan-bajgain) [inline src/pages/Home.js]: - [ ] You can make your home page looks better by providing some styling on the project like Background color and making more content. You can look for the general layout for reference
### your design
his
### Figma design
figma

File: src/components/Calculator.js [hunk 2]
```diff
@@ -19,33 +19,36 @@ const Calculator = () => {
...
+        <button className="btn" type="submit" onClick={handleClick}>1</button>
+        <button className="btn" type="submit" onClick={handleClick}>2</button>
+        <button className="btn" type="submit" onClick={handleClick}>3</button>
+        <button className="btn" id="orange" type="submit" onClick={handleClick}>+</button>
+
+        <button className="btn" type="submit" onClick={handleClick} id="zero">0</button>
+        <button className="btn" type="submit" onClick={handleClick}>.</button>
+        <button className="btn" id="orange" type="submit" onClick={handleClick}>=</button>
+
+      </div>
```
Comment (roshan-bajgain) [inline src/components/Calculator.js:51]: - [ ] I would suggest you decrease the size of the calculator and use the font family and font size to the content so It looks nicer.
### your design
cal

File: src/pages/Quote.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      const response = await fetch(API_URL);
+      const randomQuote = await response.json();
+      setQuote(randomQuote.content);
+    };
+    setTimeout(() => fetchQuote(), 60000);
+  });
+  return (
+    <h4>{ JSON.stringify(quote) }</h4>
+  );
+}
```
Comment (roshan-bajgain) [inline src/pages/Quote.js]: - [ ] Nice work on the Quotes but you can make the content bigger and and use some image, color to make it more better.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @mr-abdellah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [ ] Your project is great but you can style it and make it more beautiful.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mr-abdellah** [comment]
hello @roshan-bajgain thanks for your review, but is styling so important?

**Comment by roshan-bajgain** [comment]
@mr-abdellah Hello bro, You can make the websites more beautiful by adding some styling, You don't need to make it perfect but you can make the website looks beautiful. As it would help you in a job search when a recruiter looks at your websites, the first thing they notice is your design So In my point, It is essential and important. Thanks for getting in touch

**Comment by mr-abdellah** [comment]
Hey @roshan-bajgain thank you so much , now it's clear , because i was focusing on routing, hooks ..etc not styling , I'll style it now , thanks again !

**Review by arslanbisharat — APPROVED** [review]
Hi @mr-abdellah,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**